webpack/scenes/Hosts/ChangeContentSource/index.js

```diff
@@ -74,6 +80,13 @@ const ChangeContentSourcePage = () => {
     }
   };
 
+  const breadcrumbItems = () => {
+    const singleHost = getHostIds(urlParams.host_id).length === 1;
+    const link = singleHost ? { caption: __('Host'), url: foremanUrl(`/new/hosts/${urlParams.host_id[0]}`) } : { caption: __('Hosts'), url: foremanUrl('/hosts') };
```

Comment:
When my host id is 28, this link tries to take me to `/new/hosts/2` and not `/new/hosts/28`